decorators/captilized_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require_relative './base_decorator'
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] `Capitalize` and `Upcase` are different methods as we need to `capitalize` please 🙏🏽 use the correct method for your CapitalizeDecorator class work as expected.